### PR TITLE
Lite-23302: add translation export command

### PR DESCRIPTION
**Pull request by qarlosh**

### CODE REVIEW

File: connect/cli/core/utils.py [hunk 1]
```diff
@@ -1,9 +1,13 @@
 # -*- coding: utf-8 -*-
```
Comment (ffaraoneim) [inline connect/cli/core/utils.py:1]: These functions are specific of your plugins right ? If so it does not make sense to put them in core/utils.py

**Review by ffaraoneim — CHANGES_REQUESTED** [review]
All the rest look fine for me

File: connect/cli/core/utils.py (hunk 1 above)
Comment (qarlosh) [reply connect/cli/core/utils.py:1]: Right now they are only used by the new translation plugin, but as I saw that this same code is repeated in other plugins, I moved them to core to allow those plugins to use them. I could do this in a later PR.

**Comment by qarlosh** [comment]
As commented, I removed `set_ws_main_header()` from core.utils because is very specific for the plugin, and now `validate_output_options()` is used from all appropriate plugins.

**Review by ffaraoneim — APPROVED** [review]

**Pull request merged**